helper-functions/src/index.ts

```diff
@@ -34,5 +34,8 @@ import { IsCron } from './IsCron';
 import { SpacedColor } from './SpacedColor';
 import { HsvToHex } from "./HsvToHex";
 import { HexToHsv } from "./HexToHsv";
+import { useEffectWithPrevious } from './React/useEffectWithPrevious';
+import { findLastIndex } from './FindLastIndex'
+namespace  React {useEffectWithPrevious}
 
-export {CreateGuid, GetTextWidth, GetNodeSize, RandomColor, GetTextHeight, IsNumber, IsInteger, IsCron, SpacedColor, HsvToHex, HexToHsv}
+export {CreateGuid, GetTextWidth, GetNodeSize, RandomColor, GetTextHeight, IsNumber, IsInteger, IsCron, SpacedColor, HsvToHex, HexToHsv, React}
```

Comment:
```suggestion
export {CreateGuid, GetTextWidth, GetNodeSize, RandomColor, GetTextHeight, IsNumber, IsInteger, IsCron, SpacedColor, HsvToHex, HexToHsv, findLastIndex, React}
```
Doesn't seem like we are exporting this new function.